prometheus-monitoring/delete.sh

```diff
@@ -0,0 +1,7 @@
+kubectl delete deployments --all
+
+kubectl delete svc --all
```

Comment:
This will delete the kubernetes service also.
```
$ kubectl get svc
NAME                     CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
kubernetes               10.96.0.1       <none>        443/TCP    3d
maya-apiserver-service   10.102.10.243   <none>        5656/TCP   2h
```

Reply:
But that's fine. This is not just go script to delete those service.

Reply:
@prateekpandey14 purpose of this code to get started with prometheus monitoring. This is written only for the very basic server.You will have to make changes for your deployments accordingly.

Reply:
Right. The aim was, if someone follows the steps(ran that script), and by mistake(unknowingly) delete the kube-service itself then they will end up doing something else :D